apps/api-extractor/src/enhancers/ValidationEnhancer.ts

```diff
@@ -210,9 +210,8 @@ export class ValidationEnhancer {
           continue;
         }
 
-        localName = rootSymbol.localName;
-
         collectorEntity = collector.tryGetCollectorEntity(rootSymbol);
+        localName = collectorEntity?.nameForEmit || rootSymbol.localName;
```

Comment:
This change and the one below is a nit bug fix unrelated to this PR (I noticed it in the test case I added). See https://github.com/microsoft/rushstack/pull/3552/files#diff-f6ecec4f81407d8e726a025aa7aef71db73f5992650ddccd959c355d6a02e31bR7.